sbt-plugin/src/main/scala/akka/grpc/sbt/AkkaGrpcPlugin.scala

```diff
@@ -106,6 +106,13 @@ object AkkaGrpcPlugin extends AutoPlugin {
       (if (config == Compile || config == Test) Seq() // already supported by sbt-protoc by default
        else sbtprotoc.ProtocPlugin.protobufConfigSettings) ++
       Seq(
+        // workaround to allow using Scala 2.13 artifacts in Scala 3 projects
+        scalaBinaryVersion := {
```

Comment:
Hmm, wouldn't this interfere with other uses of `scalaBinaryVersion`? Perhaps it would be safer to move this inside `sandboxedGenerator`/`plainGenerator` below, so the effect is more focused?

Reply:
binary version is used for pulling dependencies too, not just code generation
tried moving it into `generator` methods and published locally, that doesn't work as tries to pull Scala 3 versions
```
sbt.librarymanagement.ResolveException: Error downloading com.lightbend.akka.grpc:akka-grpc-runtime_3:2.1.0-M1-4-4ca3e1aa-20210708-0938-SNAPSHOT
```

Reply:
@raboof okay, seems to work now, moved Scala binary version for generators and their dependencies
published it locally and seems to work fine with my Scala 3 project 👍